vsts_ci/standalone-binaries/continuous-build-standalone-binaries-win32.yml

```diff
@@ -13,6 +13,7 @@ steps:
       .\venv\Scripts\activate
       python -m pip install --upgrade pip
       if ("$(BUILD.SOURCEBRANCH)" -match "^refs/tags/v?[0-9]+\.[0-9]+\.[0-9]+$") { ((Get-Content -path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\__init__.py" -Raw) -replace "__AIkey__ = '.*'","__AIkey__ = '$(AI_KEY)'") | Set-Content -Path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\__init__.py" }
+      ((Get-Content -path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\telemetry.py" -Raw) -replace "PRODUCT_NAME = 'iotedgehubdev'","PRODUCT_NAME = 'iotedgehubdev-standalone'") | Set-Content -Path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\telemetry.py"
```

Comment:
is it possible to move product name into __init__.py and reference here? Since we modify ai key in init py. Better to have properties like this in the same place

Reply:
OK

Reply:
Updated